docker/envoy-oidc-filter/Dockerfile

```diff
@@ -1,3 +1,8 @@
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+
 FROM scratch
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
This is needed when calling an endpoint which has a CA signed certificate ? (without skipping server cert verification) ?

Reply:
yes, the scratch base image does not have any ca-certificates which causes to fail the container even when calling accounts.google.com